plugins/modules/lxd_container.py

```diff
@@ -547,6 +543,8 @@ def _create_instance(self):
             url_params['target'] = self.target
         if self.project:
             url_params['project'] = self.project
+        if self.type:
+            url_params['type'] = self.type
```

Comment:
Could this also break older LXD versions?

Reply:
I have not explicitly tested it, but a quick peruse of the code suggests that unexpected query parameters are ignored: https://github.com/canonical/lxd/blob/cf9bf2ac072044118dfbd9d49f2342298a52c2d2/lxd/instances_get.go#L244

Reply:
That would be good.